src/tools/kdb/rm.cpp

```diff
@@ -35,6 +35,13 @@ int RemoveCommand::execute (Cmdline const & cl)
 
 	KeySet conf;
 	Key x = cl.createKey (0);
+
+	if (x.getNamespace () == kdb::ElektraNamespace::CASCADING)
+	{
+		cerr << "Aborting: Specify a namespace for deleting a key." << endl;
+		return 12;
```

Comment:
Please update the `kdb-rm` man page with the meaning of 12 as return value

Reply:
done